tools/ratbagctl.in.in

```diff
@@ -185,6 +185,7 @@ def print_resolution(d, p, r, level):
     print(" " * level + "{}: {}dpi{}{}".format(r.index,
```

Comment:
You need to add a `{}` at the end here.

Reply:
Ah oh, totally didn't see that, thanks.

Reply:
I think this might be what was messing up the tests, can you check?

Reply:
Just saw it after I force pushed. Will check again, probably a Flake error.